### DISCUSSION TITLE
Getting "Update the Facebook SDK for iOS" on Events Manager, is that expected?

### DISCUSSION THREAD

**Discussion by appm8ker** [oddbit/flutter_facebook_app_events#294] [General]
It says to use:
pod 'FBSDKCoreKit', '~> 8.0.0'
pod 'FBSDKLoginKit', '~> 8.0.0'
pod 'FBSDKShareKit', '~> 8.0.0'
pod 'FBSDKGamingServiceKit', '~> 8.0.0'
![Screenshot 2023-05-03 at 4 15 11 PM](https://user-images.githubusercontent.com/35655046/236071341-f31f947e-7fb8-4fdf-aeda-a3975b81a5b8.png)

But this is specifically set on plugin, right?

**Comment by HarsHvA** [comment]
@renanhiramatsu Did you found any fix?

**Reply by appm8ker** [reply]
Unfortunately no. Haven't seen much problem though.

**Reply by pndychrg** [reply]
Hey did you find any resolution for your problem

**Comment by hildaylchung** [comment]
I noticed a pull request regarding Facebook SDK Update is merged to develop branch a few days ago. Would a new version be released to fix this warning?

**Comment by dohuon** [comment]
I'm still encountering the same issue. The version I'm using is facebook_app_events: ^0.19.2. 
Is there any progress?

**Reply by pndychrg** [reply]
Suffering with the same issue, any resolution found ??

**Comment by hilmirstraum** [comment]
Is there any progress to this issue?

**Comment by wildsurfer** [comment]
Facing the same issue

**Comment by Phillewis63** [comment]
I have the same issue. Is there a fix for it yet?
